test/classnames.js

```diff
@@ -76,18 +76,14 @@ test('(compat) handles all types of truthy and falsy property values as expected
 		false: false,
 		undefined: undefined,
 
-		// truthy (literally anything else):
+		// truthy
 		nonEmptyString: 'foobar',
 		whitespace: ' ',
 		function: Object.prototype.toString,
-		emptyObject: {},
-		nonEmptyObject: {a: 1, b: 2},
-		emptyList: [],
-		nonEmptyList: [1, 2, 3],
-		greaterZero: 1
```

Comment:
Removing these isn't a fix. The next assertion fails w/ these restored, which shows that this PR is a breaking change.

Reply:
Thanks, I didn't propose it as a non-breaking change. You are already doing versioned releases; this would be a new breaking release, no?

I removed those because they no longer apply as truthy - not as a fix, but a change.

I tried to keep the code style changes to a min.

Reply:
would it be better to propose this as a separate module then, like `lite`?